install-devenv.sh

```diff
@@ -36,13 +42,17 @@ curl -fsSL \
 
 echo "${sha256}  ${tmpd}/${archive}" | /usr/bin/shasum -a 256 --check --status
 
+# Question: What is the --strip-components doing here?
 tar --strip-components=1 -C "$devenv_python_root" -x -f "${tmpd}/${archive}"
 
+# Question: Is there a reason to use the SSH URL here?
 uri='git@github.com:getsentry/devenv'
 [[ $CI ]] && uri='https://github.com/getsentry/devenv.git'
 git -C "$devenv_root" clone -q --depth=1 "$uri"
 
+# Question: What does this syntax mean / do?
```

Comment:
uhhhhhh actually this is wrong and i need to fix it - it should be a substring test